src/Foundatio.Logging.Xunit/TestLoggerFactory.cs

```diff
@@ -12,7 +12,7 @@ public class TestLoggerFactory : ILoggerFactory {
             _testOutputHelper = output;
         }
 
-        public LogLevel MinimumLevel { get; set; } = LogLevel.Information;
+        public LogLevel MinimumLevel { get; set; } = LogLevel.Trace;
```

Comment:
I'd say the biggest thing is it can slow down your tests big time as everything gets that min log level. If we set the min log level to info. Then in the specific Test we can do Log.SetMinLevel<Queue>(Trace) and then we won't get trace info for caches etc.. This also makes looking through the build log much smaller.